test/lsp.py

```diff
@@ -537,6 +538,18 @@ def at_end(self):
         """
         return self.current_line_tuple is None
 
+class FileLoadStrategy(Enum):
+    Undefined = auto()
+    ProjectDirectory = auto()
+    DirectlyOpenedAndOnImport = auto()
+
+    def lsp_name(self):
+        if self == FileLoadStrategy.ProjectDirectory:
+            return 'project-directory'
+        elif self == FileLoadStrategy.DirectlyOpenedAndOnImport:
+            return 'directly-opened-and-on-import'
+        return None
```

Comment:
You could do this instead:
```suggestion
class FileLoadStrategy(Enum):
    Undefined = None
    ProjectDirectory = 'project-directory'
    DirectlyOpenedAndOnImport = 'directly-opened-and-on-import'
```
You can then access the strings through the `.value` member.